### PR TITLE
add support for RTL

### PR DESCRIPTION
**Pull request by atmonshi**
added support for `RTL` direction.
added Arabic translation.
fix `<x-blade-icon` in `resources/views/components/item.blade.php` line 44

and thank you so much for this 🙏🏽

### CODE REVIEW

**Comment by atmonshi** [comment]
screenshot for rtl page:

Screenshot 2023-08-18 at 6 31 17 PM
Screenshot 2023-08-18 at 6 19 13 PM

**Comment by saade** [comment]
Hi, thanks! What's the reason for using `@svg` instead of `<x-icon`?

**Comment by atmonshi** [comment]
> Hi, thanks! What's the reason for using `@svg` instead of `<x-icon`?

it won't work in filament default installation, throw exception: (component not found)

since filament disables the blade components for the blade icons by default.
`\Filament\Http\Middleware\DisableBladeIconComponents`

**Comment by saade** [comment]
LOL, didn't know that. Any reason for it to be working in my environment?

**Comment by atmonshi** [comment]
not sure :), did you remove 'DisableBladeIconComponents' from the middleware in `AdminPanelProvider`?

or you already published the config for blade icon and set it to true 'blade-icons.components.disabled'

**Comment by atmonshi** [comment]
ya definitely the config `blade-icons.php`

`'disabled' => false,`

**Comment by saade** [comment]
oh, I've published it to support a custom icon set in my current project. thanks for the information =)

File: resources/views/components/item.blade.php [hunk 1]
```diff
@@ -41,13 +41,13 @@ class="px-2 text-gray-500 appearance-none"
                         title="{{ __('filament-adjacency-list::adjacency-list.actions.toggle-children.label') }}"
                         x-on:click="open = !open"
                     >
-                        <x-heroicon-o-chevron-right class="w-3.5 h-3.5 transition ease-in-out duration-200" x-bind:class="{'rotate-90': open}" />
+                        @svg('heroicon-o-chevron-right','w-3.5 h-3.5 transition ease-in-out duration-200 rtl:rotate-180', ['x-bind:class' => "{'ltr:rotate-90 rtl:!rotate-90': open}"])
```
Comment (saade) [inline resources/views/components/item.blade.php]:
```suggestion
                        @svg('heroicon-o-chevron-right', 'w-3.5 h-3.5 transition ease-in-out duration-200 rtl:rotate-180', ['x-bind:class' => "{'ltr:rotate-90 rtl:!rotate-90': open}"])
```

**Review by saade — APPROVED** [review]

**Pull request merged**